src/components/Header/PointsBal.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { useSelector } from "react-redux"
+
+import { PointsBalance, NumberBalance } from "./Header.styled"
+
+const PointsBal = () => {
+    let userBalance = 0
+    const userData = useSelector(state => state.userInfo.userData)
+    if (userData) {
+        userBalance = userData.balance
```

Comment:
pls get balance from store directly

Reply:
changed like this:

let userBalance = 0
    const userAuthorized = store.getState().userInfo.token
    if (userAuthorized) {
        userBalance = store.getState().userInfo.userData.balance
    }

Reply:
Unfortunately, in this case I get an error. What should I do?

Reply:
I tried to replace let with const, but then it looks like something is changing with the scope

Reply:
Yes, I fixed everything. Thank you for the lesson))
![image](https://user-images.githubusercontent.com/96341034/189453378-fd4e166b-0db5-438f-9353-6b37a7e0ed9a.png)